enhancements/windows-containers/monitoring-windows-nodes.md

```diff
@@ -0,0 +1,182 @@
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The intent of this enhancement is to enable performance monitoring on Windows
+nodes created by Windows Machine Config Operator(WMCO) in OpenShift cluster.
+
+## Motivation
+
+Monitoring is critical to identify issues with nodes, containers running on the
+nodes. The main motivation behind this enhancement is to enable monitoring on
+the Windows nodes.
+
+### Goals
+
+As part of this enhancement, we plan to do the following:
+* Run [windows_exporter](https://github.com/prometheus-community/windows_exporter)
+  as a service on Windows nodes
+* Upgrade the windows_exporter on the Windows Nodes
+* Leverage cluster-monitoring operator that sets up Prometheus, Alertmanager
+  and other components
+
+### Non-Goals
+
+As part of this enhancement, we do not plan to do the following:
+* Integrating windows_exporter with cluster monitoring operator
+* Ship Grafana dashboards for Windows Nodes
+
+## Proposal
+
+The main idea here is to run windows_exporter as a Windows Service and let
+Prometheus instance which was provisioned as part of OpenShift install to
+collect data from windows_exporter. The metrics exposed by the windows_exporter
+will be used to display console graphs for Windows nodes.
+
+## Justification
+
+Unlike [Node exporter](https://github.com/prometheus/node_exporter) on Linux
+nodes, windows_exporter cannot run as a container on the Windows nodes since
+Windows container images contains a Windows Kernel and Red Hat has a policy not
+to ship third party kernels for support reasons. Please refer to the [WMCO
+ enhancement](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/windows-machine-config-operator.md#justification)
+for more details.
+
+### Risks and Mitigations
+
+* Running `windows_exporter` as a Windows Service, posses a risk of having
+  inadequate resources to run the service if the Windows node is overwhelmed
+  with workload containers. This can be mitigated by leveraging [priority
+   classes](https://docs.microsoft.com/en-us/windows/win32/procthread/scheduling-priorities) for
+  Windows processes. This is similar to what is being done for other [Windows
+  services](https://issues.redhat.com/browse/WINC-534).
+  
+* One of the risks with the current approach is renaming Windows metrics to
+  display pod graphs. The pod metrics for Linux come from cAdvisor. However, we
+  do not get same metrics from cAdvisor for Windows nodes. This becomes a
+  hindrance to display pod graphs by creating custom recording rules to use same
+  console queries as Linux workloads. To mitigate this, use metrics exposed by
+  the windows_exporter to display pod graphs as mentioned in the [Future
+   Plans](https://github.com/openshift/enhancements/blob/master/enhancements/windows-containers/monitoring-windows-nodes.md#future-plans)
```

Comment:
This link is a 404 for me (as are some of the other links to this enhancement below).